### PR TITLE
update llvm to a7ac120a9ad7

### PR DESCRIPTION
**Pull request by etiotto**
Ths PR updates the LLVM commit we use to `a7ac120a9ad784998a5527fc0a71b2d0fd55eccb`.

Signed-off-by: Ettore Tiotto <etiotto@ca.ibm.com>

### CODE REVIEW

**Comment by etiotto** [comment]
@jenkins-droid test this please

File: src/Dialect/ONNX/ONNX.td [hunk 1]
```diff
@@ -244,7 +244,7 @@ def ONNX_SeqType : ONNX_Type<"Seq", "Seq"> {
     $_printer << "<" << getImpl()->ElementType << ">";
   }];
   let parser = [{
```
Comment (sstamenova) [inline src/Dialect/ONNX/ONNX.td:246]: Looks like I missed one! We should convert this to not user parser anymore either.

**Review by sstamenova — APPROVED** [review]
LGTM. It would be nice to remove the last remaining parser, but that can be done separately.

File: src/Dialect/ONNX/ONNX.td (hunk 1 above)
Comment (etiotto) [reply src/Dialect/ONNX/ONNX.td:246]: OK we can do that in another PR.

**Pull request merged**